### PR TITLE
postinstall script to apply webpack shim.

### LINKED ISSUE #156: angular-lib build path doesn't work with mqtt as dependency
@nosovk Unfortunately, the published package is broken, since mqtt can't be embedded without browserification.
Do you have a solution for this issue, or know how to embed the old vendor mqtt.js?

That's the error I get when using the library in 7.0.9.
```
index.js:43 Uncaught ReferenceError: global is not defined
    at Object../node_modules/buffer/index.js (index.js:43)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/readable-stream/node_modules/safe-buffer/index.js (index.js:2)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/readable-stream/lib/_stream_readable.js (_stream_readable.js:55)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/readable-stream/readable-browser.js (readable-browser.js:1)
    at __webpack_require__ (bootstrap:79)
    at Object../node_modules/mqtt/lib/store.js (store.js:8)
    at __webpack_require__ (bootstrap:79)
```

### PR DESCRIPTION
**Pull request by AlxZchk**
Applied hack to fix #156

### CODE REVIEW

File: projects/ngx-mqtt/postinstall.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const f = '../../node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js';
+
+fs.readFile(f, 'utf8', function (err,data) {
+  if (err) {
+    return console.log(err);
```
Comment (nosovk) [inline projects/ngx-mqtt/postinstall.js:6]: change to console.err

File: projects/ngx-mqtt/postinstall.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  }
+  // at some moment angular-cli tam disabled node polyfills and stubs in webpack
+  // from:
+  // node: false
+  // to:
+  // node: {global: true}
+  const result = (data.replace(/node: false/g, "node: {global: true}"));
+  fs.writeFile(f, result, 'utf8', function (err) {
+    if (err) return console.log(err);
```
Comment (nosovk) [inline projects/ngx-mqtt/postinstall.js:15]: if script errored - do not forget to return error code different from 0, to block further npm execution

File: projects/ngx-mqtt/postinstall.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const f = '../../node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js';
```
Comment (nosovk) [inline projects/ngx-mqtt/postinstall.js:2]: provide ENV check to disable patching

**Comment by nosovk** [comment]
@sclausen could you review, merge pls

**Comment by sclausen** [comment]
So you’re okay with [that unresolved conversation](https://github.com/sclausen/ngx-mqtt/pull/158#discussion_r469439826)

**Comment by AlxZchk** [comment]
resolved

**Comment by AlxZchk** [comment]
It was resolved like 5 days ago. I forgot to press the buttons

**Pull request merged**